File: Mobile-App/app/src/main/java/com/example/happyhabits/feature_application/feature_food/data/model/FoodStatistics.kt
package com.example.happyhabits.feature_application.feature_food.data.model

import kotlinx.serialization.Serializable

@Serializable
class FoodStatistics(
    var proteinPercentage: Float,
    var carbsPercentage: Float,
    var fiberPercentage: Float,
    var fatsPercentage: Float
) {
    override fun toString(): String {
        return "FoodStatistics(proteinPercentage=$proteinPercentage, carbsPercentage=$carbsPercentage, fiberPercentage=$fiberPercentage, fatsPercentage=$fatsPercentage)"
    }
}

File: API-Server/Happy Habits App/Forms/FoodStatistics.cs
using System.Text.Json.Serialization;

namespace Happy_Habits_App.Forms
{
    public class FoodStatistics
    {
        [JsonPropertyName("proteinPercentage")]
        public float ProteinPercentage { get; set; }
        [JsonPropertyName("carbsPercentage")]
        public float CarbsPercentage { get; set; }
        [JsonPropertyName("fiberPercentage")]
        public float FiberPercentage { get; set; }
        [JsonPropertyName("fatsPercentage")]
        public float FatsPercentage { get; set; }
    }
}
